res/values-en/strings.xml

```diff
@@ -38,15 +38,15 @@
     <string name="pref_credits">Credits</string>
     <string name="pref_pime">Win10 PIME Platform</string>
 
-    <string name="pref_input">Input</string>
+    <string name="pref_input">Input Setting</string>
     <string name="pref_enable">Enable</string>
     <string name="pref_enable_summary">Enable TRIME</string>
-    <string name="pref_select">Select</string>
+    <string name="pref_select">Change Keyboard</string>
```

Comment:
I refer the text in Android Setting, and you select it the pop view title is **Change Keyboard** if you setting your Android to use english.